src/Models/FileManagement/FileManagementVariantLogData.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace GoApptiv\FileManagement\Models\FileManagement;
+
+class FileManagementVariantLogData
+{
+    private $uuid;
+    private $variantType;
+    private $status;
+    private $errors;
+    
+    /**
+     *
+     * @param string $uuid
+     * @param string $variantType
+     * @param string $status
+     * @param string $errors
+     *
+     */
+    public function __construct(
+        string $uuid = null,
```

Comment:
Any particular reason to keep uuid as null?

Reply:
done.